src/main/java/com/onaio/steps/helper/CursorHelper.java

```diff
@@ -110,4 +111,14 @@
         cursor.close();
         return participants;
     }
+
+    @SuppressLint("Range")
+    public String getString(Cursor cursor, String column, String defaultValue) {
```

Comment:
I don't think that this actually solves the problem. It just prevents the crash by masking the error. Kindly fix the actual issue. Android does not throw randomly throw cursor errors

Reply:
fixed.